Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiRequestBus.h

```diff
@@ -22,6 +22,9 @@ namespace AzToolsFramework::ViewportUi
     using SwitcherId = IdType<struct SwitcherIdType>;
     using TextFieldId = IdType<struct TextFieldIdType>;
 
+    //! Callback function for viewport UI back button.
+    using ViewportUIBackButtonCallback = AZStd::function<void()>;
```

Comment:
nit: `ViewportUiBackButtonCallback` (lowercase i in `Ui`).

Reply:
fixed 👍